views/editor/bramble.html

```diff
@@ -76,6 +77,36 @@
             -webkit-animation: none;
```

Comment:
Not part of this patch, but can you flip the order of this `animation` and `-webkit-animation` so the non-prefixed is first?